src/app/components/TodoList.js

```diff
@@ -0,0 +1,119 @@
+import TodoApi from '../services/TodoApi';
+import SelectColor from './SelectColor';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+export default class TodoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: []
+    }
+  }
+  componentDidMount() {
+    this.reloadList()
+  }
+
+  reloadList(){
+    new TodoApi().readFetch().then(
+      (result) => {
+        this.setState({
+          isLoaded: true,
+          items: result.items
+        });
+      },
+      (error) => {
+        this.setState({
+          isLoaded: true,
+          error
+        });
+      }
+    )
+  }
+
+  readNewItem(){
+    let newItem = document.getElementById('todo')
```

Comment:
prefer using const for static variables

Reply:
you should try to never use DOM when manipulating elements in React or any other framework. What you could do is use Refs
https://reactjs.org/docs/refs-and-the-dom.html#gatsby-focus-wrapper